resources/puppetlabs/lein-ezbake/template/global/ext/cli/stop.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set +e
+
+pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
```

Comment:
@haus I went ahead and just added `procps` (not conditioning for `procps-ng` where available) in 43d35b0.  Worked fine for CentOS 7 despite it having `procps-ng` instead.  I tested this on some other distros and it seemed fine - CentOS 6, Debian 8, Ubuntu 12.04, Ubuntu 16.04, SLES 11/12.  If it proves to be a problem, we could make it conditional later.  Sound okay?